### PR TITLE
Updating doc to use new quickstart variable names

### PR DESCRIPTION
**Pull request by gberl002**
Quickstart related variable names were changed in quickstart cleanup, this updates the doc to reflect the new variable names.

### CODE REVIEW

File: docusaurus/docs/learn/quickstarts/network/local-no-docker.md [hunk 1]
```diff
@@ -58,16 +58,22 @@ ziti-controller started as process id: 1286. log located at: /home/vagrant/.ziti
 
 ### Verify the Controller is Running
 
-Assuming you have sourced the script, you will have an environment variable set named `$ZITI_EDGE_CONTROLLER_API`. After
-the controller has started, your controller should be listening at that hostname:port combination. You can see what your
-value is set to by running `echo $ZITI_EDGE_CTRL_ADVERTISED`. This variable defaults to: `$(hostname):1280`. Make sure the
-controller is on and listening and then start the edge router. 
+Assuming you have sourced the OpenZiti quickstart script, you will have environment variables set named
```
Comment (dovholuknf) [inline docusaurus/docs/learn/quickstarts/network/local-no-docker.md]: > OpenZiti quickstart script

maybe also link to it? or something like that?

Comment (dovholuknf) [reply docusaurus/docs/learn/quickstarts/network/local-no-docker.md]: also technically it's probably: 'assuming you ran the expressInstall function' too ... just sourcing the script won' tdo it?

**Review by dovholuknf — DISMISSED** [review]
lgtm. maybe make a few updates here/there

File: docusaurus/docs/learn/quickstarts/network/local-no-docker.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Running the latest version of Ziti locally is as simple as running this one comm
 
 This script will perform an 'express' install of Ziti which does the following:
 
-* download the latest version of the Ziti components (`ziti`, `ziti-controller`, `ziti-edge-router`, `ziti-tunnel`)
+* download the latest version of the Ziti components (`ziti`, `ziti-controller`, `ziti-router`, `ziti-tunnel`)
```
Comment (dovholuknf) [inline docusaurus/docs/learn/quickstarts/network/local-no-docker.md]: now-a-days we don't need `ziti-controller` and `ziti-router` nor `ziti-tunnel`. can you find/replace those with just "download the latest `ziti` binary"?

File: docusaurus/docs/learn/quickstarts/network/local-no-docker.md [hunk 3]
```diff
@@ -58,16 +58,22 @@ ziti-controller started as process id: 1286. log located at: /home/vagrant/.ziti
 
 ### Verify the Controller is Running
 
-Assuming you have sourced the script, you will have an environment variable set named `$ZITI_EDGE_CONTROLLER_API`. After
-the controller has started, your controller should be listening at that hostname:port combination. You can see what your
-value is set to by running `echo $ZITI_EDGE_CTRL_ADVERTISED`. This variable defaults to: `$(hostname):1280`. Make sure the
-controller is on and listening and then start the edge router. 
+After running `expressInstall`, you will have environment variables set named `ZITI_CTRL_EDGE_ADVERTISED_ADDRESS` and
```
Comment (dovholuknf) [inline docusaurus/docs/learn/quickstarts/network/local-no-docker.md:61]: "After running `expressInstall`, you will have environment variables set"

Is that a change? Can we guarantee the script is able to export those variables? I'm not a bash ninja, but I didn't think we could do that?

Comment (gberl002) [reply docusaurus/docs/learn/quickstarts/network/local-no-docker.md:61]: I'm not sure if it's true for all environments but it works in Mac, do you know if the env vars are set or not in Linux (WSL)?

File: docusaurus/docs/learn/quickstarts/network/local-no-docker.md (hunk 2 above)
Comment (gberl002) [reply docusaurus/docs/learn/quickstarts/network/local-no-docker.md]: Will fix

**Comment by gberl002** [comment]
Closing to create a new branch and PR, cannot merge unsigned commits from `main`

**Pull request closed without merging**